src/index.jsx

```diff
@@ -13,48 +13,13 @@ import {
 } from '@edx/frontend-platform';
 import { IntlProvider } from 'react-intl';
 
-import {
-  faFacebookSquare,
-  faTwitterSquare,
-  faLinkedin,
-  faRedditSquare,
-} from '@fortawesome/free-brands-svg-icons';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-
-import SiteFooter, { messages as footerMessages } from '@edx/frontend-component-footer';
+import Footer, { messages as footerMessages } from '@edx/frontend-component-footer';
```

Comment:
`footerMessages` is an unused variable?

Reply:
^ It is used further down in the `initialize` script